internal/node/test/BUILD.bazel

```diff
@@ -368,6 +377,7 @@ jasmine_node_test(
     data = [":expand_variables.out"],
     templated_args = [
         "$(rootpath :expand_variables.out)",
+        "$(execpath :expand_variables.out)",
```

Comment:
I decided to also test the `$@` expansion for the args and env vars, though it already must work for the test to succeed, so I could leave it out if you prefer :man_shrugging: